test/src/timeout_tests.erl

```diff
@@ -0,0 +1,131 @@
+%% The contents of this file are subject to the Mozilla Public License
+%% Version 1.1 (the "License"); you may not use this file except in
+%% compliance with the License. You may obtain a copy of the License
+%% at http://www.mozilla.org/MPL/
+%%
+%% Software distributed under the License is distributed on an "AS IS"
+%% basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+%% the License for the specific language governing rights and
+%% limitations under the License.
+%%
+%% The Original Code is RabbitMQ.
+%%
+%% The Initial Developer of the Original Code is GoPivotal, Inc.
+%% Copyright (c) 2007-2015 Pivotal Software, Inc.  All rights reserved.
+%%
+-module(timeout_tests).
+
+-compile([export_all]).
+
+-export([all_tests/0]).
+
+-import(rabbit_tests,[default_options/0, control_action/2, control_action/3,
```

Comment:
please don't import functions that are not going the be used later: `default_options/0` and `expand_options/2` are not used.